mkdocs.yml

```
@@ -14,6 +14,17 @@ nav:
   - Welcome: "index.md"
   - Data Models: "datamodels.md"
   - Data dictionaries: "dictionaries.md"
+  - mr_proc: 
```

```suggestion
  - MR Proc user guide: 
```
I think that'd be clearer in the nav bar